gnome-initial-setup/pages/account/gis-account-page-local.c

```diff
@@ -44,6 +44,9 @@
 #define SHARED_ACCOUNT_USERNAME "shared"
 #define SHARED_ACCOUNT_FULLNAME "Shared Account"
 
+#define VENDOR_ACCOUNT_GROUP "Account"
+#define VENDOR_ACCOUNT_DEFAULT_AVATAR_KEY "Default Avatar"
```

Comment:
And this should be `default-avatar`.

Reply:
In the code we committed to add the extra "faces" directory, we called the key `Faces` (still part of this commit: https://github.com/endlessm/gnome-initial-setup/commit/f13afa435cd70e3e7e95a4601109a721b049c10a).
Does that mean that I should also rename that?

Reply:
I think we should. But like in my previous comment, maybe Will can untie this.